app/src/main/AndroidManifest.xml

```diff
@@ -22,7 +22,8 @@
     android:icon="@mipmap/kiwix_icon"
     android:label="@string/app_name"
     android:supportsRtl="true"
-    android:theme="@style/AppTheme">
+    android:theme="@style/AppTheme"
+      android:fullBackupContent="@xml/backup_descriptor">
```

Comment:
I don't think there is a file with the name **backup_descriptor**. This might be the reason for build failure.

Reply:
I suggest refraining from changing stuff in AndroidManifest.xml. Before submitting a PR try running the app on a device locally to check if everything works fine.